installation/nuget.md

```diff
@@ -12,6 +12,8 @@ position: 1
 
 This article explains how to add the private Telerik NuGet feed to your system. You can use it to obtain the Telerik UI for Blazor components.
 
+@[template](/_contentTemplates/common/get-started.md#nuget-update-note)
```

Comment:
consider adding this template to the nuget feed setup section of the getting started articles (both tutorials and maybe even the "what you need" one).

Reply:
Done